config/credentials.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    username: 'standard_user',
+    password: 'secret_sauce',
```

Comment:
that is good, so if you have different types of users you can have separate objects. For example Standart user, Premuim user etc.